.github/file-paths.yaml

```diff
@@ -97,6 +99,7 @@ e2e_ci: &e2e_ci
   - ".github/actions/prepare-cypress/**"
   - ".github/actions/run-snowplow-micro/**"
   - ".github/workflows/e2e-tests.yml"
+  - ".github/workflows/e2e-cross-version.yml"
```

Comment:
Why are we running ALL e2e tests whenever we change the cross-version yaml?
Unless I'm missing something, this doesn't make sense to me.

What you want instead is to run only cross-version tests whenever this file changes.